<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1673</width>
    <height>822</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_videoframe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1161</width>
      <height>701</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="text">
     <string>video_player</string>
    </property>
   </widget>
   <widget class="QLabel" name="slider_videoframe_label">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>710</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>current_frame/total_frame</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1161</width>
      <height>701</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(126, 181, 253)</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play_or_stop">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>640</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>709</y>
      <width>1151</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QSlider" name="slider_videoframe">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_show_object_location">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>620</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>顯示 Actor 的位置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_remove_ego_id">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>10</y>
      <width>314</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Ego ID</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_set_new_ego_id_range">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>40</y>
      <width>314</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Set New Ego Frame Range</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>120</y>
      <width>331</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>311</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_ego_id"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5aa6ff;&quot;&gt;Actor ID: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_other_actor_id"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5aa6ff;&quot;&gt;Ego ID  : &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_combobox_ego_id">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>%/%</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_combobox_other_actor_id">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>%/%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>250</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="pushButton_apply_actor_filter">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string> Apply PET or distance Filter</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_pet">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>105</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PET</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_pet_min">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_pet_max">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_distance">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>105</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>distance</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_distance_min">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_distance_max">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-10000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_quick_setup">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Quick Setup</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_next_actor">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>580</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Next Actor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_0">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>0</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_1">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>60</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Ego 路口直行，遇到對向車左轉 &lt;br/&gt;{Car, Truck, Motor/Bike} &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>直行 + 左轉</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_2">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>120</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>左轉 + 直行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_3">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>180</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>並行( 機車加速通過）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_4">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>240</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>並行( 機車等速通過）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_5">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>300</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>並行( 機車減速     ）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_6">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>360</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>繞過前方車輛</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_7">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>420</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>被 右側 cut-in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_8">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>480</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>被 左側 cut-in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_9">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>540</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>右轉 ＋ 機車直行/待轉</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_10">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>600</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>左轉 + 待轉機車</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_11">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>660</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>右轉 + 行人通過</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_12">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>720</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>左轉 ＋ 行人通過</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_notice_on">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>680</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>開啟label 描述</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_label_info">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>370</y>
      <width>331</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_speed">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>770</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>780</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#5aa6ff;&quot;&gt;Speed: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_make_video_gif">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>740</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>輸出影像</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_this_ego_done">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>70</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Mark This Ego Done</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_show_unlabeled_ego">
    <property name="geometry">
     <rect>
      <x>1350</x>
      <y>70</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show unalabeled Ego</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_label_13">
    <property name="geometry">
     <rect>
      <x>1520</x>
      <y>770</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>不確定</string>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>label_videoframe</zorder>
   <zorder>slider_videoframe_label</zorder>
   <zorder>pushButton_play_or_stop</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>pushButton_show_object_location</zorder>
   <zorder>pushButton_remove_ego_id</zorder>
   <zorder>pushButton_set_new_ego_id_range</zorder>
   <zorder>frame</zorder>
   <zorder>frame_2</zorder>
   <zorder>pushButton_next_actor</zorder>
   <zorder>pushButton_label_0</zorder>
   <zorder>pushButton_label_1</zorder>
   <zorder>pushButton_label_2</zorder>
   <zorder>pushButton_label_3</zorder>
   <zorder>pushButton_label_4</zorder>
   <zorder>pushButton_label_5</zorder>
   <zorder>pushButton_label_6</zorder>
   <zorder>pushButton_label_7</zorder>
   <zorder>pushButton_label_8</zorder>
   <zorder>pushButton_label_9</zorder>
   <zorder>pushButton_label_10</zorder>
   <zorder>pushButton_label_11</zorder>
   <zorder>pushButton_label_12</zorder>
   <zorder>pushButton_label_notice_on</zorder>
   <zorder>tableWidget_label_info</zorder>
   <zorder>comboBox_speed</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton_make_video_gif</zorder>
   <zorder>pushButton_this_ego_done</zorder>
   <zorder>pushButton_show_unlabeled_ego</zorder>
   <zorder>pushButton_label_13</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
